x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -175,6 +178,13 @@ function processEdgeLineSegments(
       throw new Error();
     }
 
+    const parentTime = eventTimestamp(parent);
```

Comment:
:grey_question: A couple more descriptive names like `parent*Start*Time` and `child*Start*Time` might help give context to the `elapsedTime`